manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.DataSpell
+PackageVersion: 2022.3.2
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
```

```suggestion
- silent
AppsAndFeaturesEntries:
- DisplayVersion: 223.8617.58
  ProductCode: DataSpell 2022.3.2
```

To match registry entries

---

Thanks